### PR TITLE
Yanstory

### PR DESCRIPTION
**Pull request by zhaoyanthu**
Add new yanstory

### CODE REVIEW

**Comment by zhaoyanthu** [comment]
This is good

File: Chinese/大话西游.md [hunk 1]
```diff
@@ -154,4 +154,6 @@
 
 [作夢](dream/dream.md)
 
-[你给我站住](stop/stop.md)
\ No newline at end of file
+[你给我站住](stop/stop.md)
+
+[Gao Shen Me](Gaoshenme/gaoshenme.md)
```
Comment (casey-collab) [inline Chinese/大话西游.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: Chinese/gaoshenme/gaoshenme.md [hunk 2]
```diff
@@ -0,0 +1 @@
+shi zai gao bu dong zhe shi zai gan ma. Hao Dou Bi.
```
Comment (casey-collab) [inline Chinese/gaoshenme/gaoshenme.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by zhaoyanthu** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by zhaoyanthu** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :dart:

**Pull request merged**